docs/content/en/docs/installation-configuration/authentication.md

```diff
@@ -0,0 +1,253 @@
+OIDC_CLIENT_SECRET: "SOME_CLIENT_SECRET"
+OIDC_CLIENT_ID: "SOME_CLIENT_ID"
+OIDC_SKIP_TLS_VERIFY: true
+{{< /tab >}}
+{{< /tabpane >}}
+
+## Setting up Authentik with SOARCA
+
+Next, we need to obtain variables such as `OIDC_ISSUER` etc. This section will describe how we can setup [Authentik](https://goauthentik.io/).
+
+### Spinning up Authentik
+
+Instruction and docker-compose on how to bundle SOARCA with Authentik will come!
+
+### Making an authentication provider
+
+In Authentik first setup a provider. An example configuration is given below:
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/setup-provider.png)
+
+Next, we need to set in the advanced protocol settings the token expiration lifetime needs to be changed to 8 eight hours.
+
+{{% alert title="Warning" color="warning" %}}
+We use an token lifetime of 8 hours, since the [SOARCA-GUI](https://github.com/COSSAS/SOARCA-GUI) uses this token for client validation. As we do not want the user to login every so minute. It is advised to set this to 8 hours.
+{{% /alert %}}
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/change-lifetime.png)
+
+Endpoints for the auth provider can also be found here:
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/endpoints.png)
+
+### Making an application
+
+Next create an application as shown in the picture below. Add the earlier made provider to this application.
+
+![core](/SOARCA/images/installation_configuration/authentik_setup/setting-application.png)
+
+### Setting the authentication provider
+
+Next, under `providers` -> `soarca-auth-provider` -> `edit` we can find the following overview:
+
+Here we can find the:
+
+- Client ID: `some random stuff`
+- Client Secret': `some other random stuff`
+- Redirect Url': Optional, should be set when using for the SOARCA-GUI explained [here]
+
+Set these variables in the environment variables settings, for example:
+
+{{% alert title="Note" color="primary" %}}
+We only use the Authentik integration for validation on the SOARCA side. As such, only the `OIDC_CLIENT_ID` is required here. For the SOARCA-GUI, we would need the `OIDC_CLIENT_SECRET`.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+It is not advised to run Authentik like this! Please setup TLS certificates in a real environment and set the `OIDC_SKIP_TLS_VERIFY` to `false`.
+{{% /alert %}}
+
+{{< tabpane langEqualsHeader=false  >}}
+{{< tab header="`.env`" lang="txt" >}}
+AUTH_ENABLED: true  
+OIDC_ISSUER: "https://authentikuri/application/o/does-providing-for-soarca/"
+OIDC_CLIENT_ID: "WxUcBMGZdI7c0e5oYp6mYdEd64acpXSuWKh8zBH5"
+OIDC_SKIP_TLS_VERIFY: true
+{{< /tab >}}
+
+{{< tab header="`bash`" lang="bash" >}}
+export AUTH_ENABLED=true
+export OIDC_ISSUER="https://authentikuri:9443/application/o/does-providing-for-soarca/"
+export OIDC_CLIENT_ID="WxUcBMGZdI7c0e5oYp6mYdEd64acpXSuWKh8zBH5"
+export OIDC_SKIP_TLS_VERIFY=true
+{{< /tab >}}
+{{< /tabpane >}}
```

Comment:
we could use this style of init which I like better then the one we use now:
see: https://docs.opencti.io/latest/deployment/installation/#configure-the-environment
```
(cat << EOF
OPENCTI_ADMIN_EMAIL=admin@opencti.io
OPENCTI_ADMIN_PASSWORD=ChangeMePlease
OPENCTI_ADMIN_TOKEN=$(cat /proc/sys/kernel/random/uuid)
OPENCTI_BASE_URL=http://localhost:8080
MINIO_ROOT_USER=$(cat /proc/sys/kernel/random/uuid)
MINIO_ROOT_PASSWORD=$(cat /proc/sys/kernel/random/uuid)
RABBITMQ_DEFAULT_USER=guest
RABBITMQ_DEFAULT_PASS=guest
ELASTIC_MEMORY_SIZE=4G
CONNECTOR_HISTORY_ID=$(cat /proc/sys/kernel/random/uuid)
CONNECTOR_EXPORT_FILE_STIX_ID=$(cat /proc/sys/kernel/random/uuid)
CONNECTOR_EXPORT_FILE_CSV_ID=$(cat /proc/sys/kernel/random/uuid)
CONNECTOR_IMPORT_FILE_STIX_ID=$(cat /proc/sys/kernel/random/uuid)
CONNECTOR_EXPORT_FILE_TXT_ID=$(cat /proc/sys/kernel/random/uuid)
CONNECTOR_IMPORT_DOCUMENT_ID=$(cat /proc/sys/kernel/random/uuid)
SMTP_HOSTNAME=localhost
EOF
) > .env
```

and 

`export $(cat .env | grep -v "#" | xargs)`